packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -981,4 +1062,89 @@ class AndroidCameraCameraX extends CameraPlatform {
     return proxy.createQualitySelector(
         videoQuality: videoQuality, fallbackStrategy: fallbackStrategy);
   }
+
+  // Methods for configuring auto-focus and auto-exposure:
+
+  /// Starts a foucs and metering action.
+  ///
+  /// This method will merge the current action's metering points with that
+  /// specified such that only one auto-exposure metering point and one
+  /// auto-focus metering point will be part of the action. This method would
+  /// also accommodate at most one auto-white balance metering point, but that
+  /// mode is currently unused in this plugin.
+  ///
+  /// Thus, if [meteringPoint] is non-null, this action includes:
+  ///   * metering points and their modes previously added to
+  ///     [currentFocusMeteringAction] that do not share a metering mode with
+  ///     [meteringPoint] and
+  ///   * [meteringPoint] with the specified [meteringMode].
+  /// If [meteringPoint] is null, this action includes only metering points and
+  /// their modes previously added to [currentFocusMeteringAction] that do not
+  /// share a metering mode with [meteringPoint]. If there are no such metering
+  /// points, then the previously enabled focus and metering actions will be
+  /// canceled.
+  Future<void> _startFocusAndMeteringFor(
+      {required Point<double>? meteringPoint,
```

Comment:
nit: Maybe rename to `point`, given that this isn't a `MeteringPoint`, and that we later use (what is currently named) `meteringPoint` to make a `MeteringPoint`